# EGAPx Tests

## EGAPx v0.3.2

## WW (assembly & RNA) & ZQ (proteins) tests

#### qsub run_EGAPx_v0.3.2_HPC.sh EGAPx_v0.3.2/D_ambigua/inputs_3_WW_ZQ.txt
##### job 

#### qsub run_EGAPx_v0.3.2_HPC.sh EGAPx_v0.3.2/D_arenata/inputs_2_WW_ZQ.txt
##### job 

#### qsub run_EGAPx_v0.3.2_HPC.sh EGAPx_v0.3.2/D_catawba/inputs_3_WW_ZQ.txt
##### job 

#### qsub run_EGAPx_v0.3.2_HPC.sh EGAPx_v0.3.2/D_dentifera/inputs_3_WW_ZQ.txt
##### job 

#### qsub run_EGAPx_v0.3.2_HPC.sh EGAPx_v0.3.2/D_magniceps/inputs_WW_ZQ.txt
##### job 

#### qsub run_EGAPx_v0.3.2_HPC.sh EGAPx_v0.3.2/D_obtusa/inputs_v2_WW_ZQ.txt
##### job 

#### qsub run_EGAPx_v0.3.2_HPC.sh EGAPx_v0.3.2/D_parvula/inputs_2_WW_ZQ.txt
##### job 

#### qsub run_EGAPx_v0.3.2_HPC.sh EGAPx_v0.3.2/D_pulex/inputs_BEL2_WW_ZQ.txt
##### job 

#### qsub run_EGAPx_v0.3.2_HPC.sh EGAPx_v0.3.2/D_pulex/inputs_KAP106_NMP_1.3_WW_ZQ.txt
##### job 

#### qsub run_EGAPx_v0.3.2_HPC.sh EGAPx_v0.3.2/D_pulex/inputs_PA42_5_WW_ZQ.txt
##### job 

#### qsub run_EGAPx_v0.3.2_HPC.sh EGAPx_v0.3.2/D_pulex/inputs_STM2_asex_1.3_WW_ZQ.txt
##### job 

#### qsub run_EGAPx_v0.3.2_HPC.sh EGAPx_v0.3.2/S_vetulus/inputs_WW_ZQ.txt
##### job 

#### qsub run_EGAPx_v0.3.2_HPC.sh EGAPx_v0.3.2/D_mitsukuri/inputs_1_SZH4_WW_ZQ.txt
##### job

# To-do: double check inputs
#### qsub run_EGAPx_v0.3.2_HPC.sh EGAPx_v0.3.2/D_magna/inputs_NIES_NCBI.txt
##### job


## WW tests

#### qsub run_EGAPx_v0.3.2_HPC.sh test/EGAPx_v0.3.2/D_ambigua/inputs_3_WW.txt
##### job 1071029
COMPLETED

#### qsub run_EGAPx_v0.3.2_HPC.sh test/EGAPx_v0.3.2/D_arenata/inputs_2_WW.txt
##### job 1075892
COMPLETED

#### qsub run_EGAPx_v0.3.2_HPC.sh test/EGAPx_v0.3.2/D_catawba/inputs_3_WW.txt
##### job 1075893
COMPLETED

#### qsub run_EGAPx_v0.3.2_HPC.sh test/EGAPx_v0.3.2/D_dentifera/inputs_3_WW.txt
##### job 1075894
COMPLETED

#### qsub run_EGAPx_v0.3.2_HPC.sh test/EGAPx_v0.3.2/D_magniceps/inputs_WW.txt
##### job 1071026
COMPLETED

#### qsub run_EGAPx_v0.3.2_HPC.sh test/EGAPx_v0.3.2/D_obtusa/inputs_v2_WW.txt
##### job 1071027
COMPLETED

#### qsub run_EGAPx_v0.3.2_HPC.sh test/EGAPx_v0.3.2/D_parvula/inputs_2_WW.txt
##### job 1071025
COMPLETED

#### qsub run_EGAPx_v0.3.2_HPC.sh test/EGAPx_v0.3.2/D_pulex/inputs_BEL2_WW.txt
##### job 1075895
COMPLETED

#### qsub run_EGAPx_v0.3.2_HPC.sh test/EGAPx_v0.3.2/D_pulex/inputs_KAP106_NMP_1.3_WW.txt
##### job 1070991
FAILED
#### qsub resume_EGAPx_v0.3.2_HPC.sh test/EGAPx_v0.3.2/D_pulex/inputs_KAP106_NMP_1.3_WW.txt
##### job 1081931
COMPLETED

#### qsub run_EGAPx_v0.3.2_HPC.sh test/EGAPx_v0.3.2/D_pulex/inputs_PA42_5_WW.txt
##### job 1070992
FAILED
#### qsub resume_EGAPx_v0.3.2_HPC.sh test/EGAPx_v0.3.2/D_pulex/inputs_PA42_5_WW.txt
##### job 1081943
COMPLETED

#### qsub run_EGAPx_v0.3.2_HPC.sh test/EGAPx_v0.3.2/D_pulex/inputs_STM2_asex_1.3_WW.txt
##### job 1070993
FAILED
#### qsub resume_EGAPx_v0.3.2_HPC.sh test/EGAPx_v0.3.2/D_pulex/inputs_STM2_asex_1.3_WW.txt
##### job 1081962
COMPLETED

#### qsub run_EGAPx_v0.3.2_HPC.sh test/EGAPx_v0.3.2/S_vetulus/inputs_WW.txt
##### job 1071024
COMPLETED

#### qsub run_EGAPx_v0.3.2_HPC.sh test/EGAPx_v0.3.2/D_mitsukuri/inputs_1_SZH4_WW.txt
##### job 1082971
COMPLETED


## EGAPx v0.3.1

## Small tests

#### qsub run_EGAPx_v0.3.1_HPC.sh test/EGAPx_v0.3/D_pulex/inputs_KAP4_NCBI_small.txt
##### job 1026412
Dec 11 21:34:43 ..... finished successfully

#### qsub run_EGAPx_v0.3.1_HPC.sh test/EGAPx_v0.3/D_pulex/inputs_KAP4_NCBI_medium.txt
##### job 1026414 -> https://github.com/ncbi/egapx/issues/66
Dec-11 17:04:37.683 [TaskFinalizer-4] ERROR nextflow.processor.TaskProcessor - Error executing process > 'egapx:rnaseq_short_plane:star:run_star (1)'
##### job 1026685
Dec-11 22:47:32.101 [TaskFinalizer-6] ERROR nextflow.processor.TaskProcessor - Error executing process > 'egapx:rnaseq_short_plane:star:run_star (3)'

#### qsub run_EGAPx_v0.3.1_HPC.sh test/EGAPx_v0.3/D_pulex/inputs_KAP4_NCBI_ms.txt
##### job 1026611
Dec 12 00:32:02 ..... finished successfully

#### qsub run_EGAPx_v0.3.1_HPC.sh test/EGAPx_v0.3/D_pulex/inputs_KAP4_NCBI_SRA_small.txt
##### job 1026212
Dec 11 18:56:53 ..... finished successfully

#### qsub run_EGAPx_v0.3.1_HPC.sh test/EGAPx_v0.3/D_pulex/inputs_KAP4_NCBI_SRA_medium.txt
##### job 1026220
Dec-11 14:33:29.811 [TaskFinalizer-4] ERROR nextflow.processor.TaskProcessor - Error executing process > 'egapx:rnaseq_short_plane:star:run_star (1)'
##### job 1026686
Dec-11 22:45:11.921 [TaskFinalizer-6] ERROR nextflow.processor.TaskProcessor - Error executing process > 'egapx:rnaseq_short_plane:star:run_star (1)'

### KAP106 NCBI, ZQ, & WW tests

#### qsub run_EGAPx_v0.3.1_HPC.sh D_pulex/inputs_KAP106_NCBI_SRA.txt 
##### job 984000
ERROR -> https://github.com/ncbi/egapx/issues/57
Nov-23 22:20:36.567 [TaskFinalizer-5] ERROR nextflow.processor.TaskProcessor - Error executing process > 'egapx:rnaseq_short_plane:star:run_star (1)'

#### qsub run_EGAPx_v0.3.1_HPC.sh D_pulex/inputs_KAP106_NCBI_SRA_WW.txt 
##### job 984001
ERROR
Nov-23 22:13:37.039 [TaskFinalizer-8] ERROR nextflow.processor.TaskProcessor - Error executing process > 'egapx:rnaseq_short_plane:star:run_star (2)'

#### qsub run_EGAPx_v0.3.1_HPC.sh D_pulex/inputs_KAP106_ZQ_SRA.txt 
##### job 984002
ERROR
Nov-24 00:16:54.675 [TaskFinalizer-5] ERROR nextflow.processor.TaskProcessor - Error executing process > 'egapx:rnaseq_short_plane:star:run_star (1)'

#### qsub run_EGAPx_v0.3.1_HPC.sh D_pulex/inputs_KAP106_ZQ_SRA_WW.txt 
##### job 984003
ERROR
Nov-23 23:32:25.927 [TaskFinalizer-8] ERROR nextflow.processor.TaskProcessor - Error executing process > 'egapx:rnaseq_short_plane:star:run_star (2)'

#### qsub run_EGAPx_v0.3.1_HPC.sh D_pulex/inputs_KAP4_NCBI_SRA.txt 
##### job 984005
ERROR
ERROR ~ Error executing process > 'egapx:rnaseq_short_plane:star:run_star (1)'
Nov-29 17:37:06.362 [TaskFinalizer-2] ERROR nextflow.processor.TaskProcessor - Error executing process > 'egapx:rnaseq_short_plane:star:run_star (1)'

#### qsub run_EGAPx_v0.3.1_HPC.sh D_pulex/inputs_KAP4_NCBI_SRA_WW.txt 
##### job 984006
ERROR
ERROR ~ Error executing process > 'egapx:rnaseq_short_plane:star:run_star (2)'
Nov-29 20:49:43.965 [TaskFinalizer-6] ERROR nextflow.processor.TaskProcessor - Error executing process > 'egapx:rnaseq_short_plane:star:run_star (1)'

### Software test

#### qsub run_EGAPx_v0.3_HPC.sh test/EGAPx_v0.2/D_pulex/inputs_KAP106_NCBI_SRA_RNA.txt
##### job 983971
SUCCEEDED
Nov 21 02:07:27 ..... finished successfully

#### qsub run_EGAPx_v0.3_HPC.sh test/EGAPx_v0.1/inputs_D_farinae.txt
##### job 982938
SUCCEEDED
Completed at: 20-Nov-2024 02:23:00
Duration    : 1m 47s
CPU hours   : 4.8 (100% cached)
Succeeded   : 0
Cached      : 83

### KAP106 Isoseq test sets

#### Primary tests

#### qsub run_EGAPx_v0.3.1_HPC.sh test/EGAPx_v0.2/D_lumholtzi/inputs_lumholtzi_WW_RNA.txt
##### job 980822 -> run0
ERROR 
-> https://github.com/ncbi/egapx/issues/52
-> https://github.com/ncbi/egapx/issues/61
-> https://github.com/ncbi/fcs
-> error [nextflow.exception.ProcessFailedException]: Process `egapx:annot_proc_plane:gnomon_biotype:run_gnomon_biotype` terminated with an error exit status (3)
-> ERROR ~ Error executing process > 'egapx:annot_proc_plane:gnomon_biotype:run_gnomon_biotype'
-> Error: (106.16) Application's execution failed (CException::eUnknown) Too many protein hits to proks (GP-23178)
##### job 982926
ERROR
-> Error: (106.16) Application's execution failed (CException::eUnknown) Too many protein hits to proks (GP-23178)

#### qsub run_EGAPx_v0.3.1_HPC.sh test/EGAPx_v0.2/D_lumholtzi/inputs_lumholtzi_WW.txt
##### job 980823
ERROR
-> error [nextflow.exception.ProcessFailedException]: Process `egapx:rnaseq_short_plane:star:run_star (1)` terminated with an error exit status (3)
-> [99/772862] NOTE: Process `egapx:rnaseq_short_plane:star:run_star (1)` terminated with an error exit status (3) -- Execution is retried (3)
-> [E::sam_parse1] CIGAR and query sequence are of different length
  [W::sam_read1_sam] Parse error at line 53
  samtools view: error reading file "-"
  Traceback (most recent call last):
    File "/img/gp/bin/star-with-filter", line 32, in <module>
      subprocess.run(main_cmd, shell=True, check=True)
    File "/usr/lib/python3.8/subprocess.py", line 516, in run
      raise CalledProcessError(retcode, process.args,
  subprocess.CalledProcessError: Command 'set -euo pipefail; /img/gp/bin//../third-party/STAR/bin/Linux_x86_64/STAR --alignSJoverhangMin 8 --outFilterMultimapNmax 200 --outFilterMismatchNmax 50 --runThreadN 31 --genomeLoad NoSharedMemory --outSAMtype SAM --outSAMattributes NH HI AS nM NM MD jM jI XS MC --outSAMprimaryFlag AllBestScore --outFilterMultimapScoreRange 50 --seedSearchStartLmax 15 --limitOutSAMoneReadBytes 1000000 --outSJtype None --genomeDir D.lumholtzi_format.index --outFileNamePrefix wrkarea/STAR.116017757157768zK0EW/D.lumholtzi_format-D.lumholtzi_isoseq_coding/D.lumholtzi_format-D.lumholtzi_isoseq_coding- --outStd SAM --readFilesIn ./D.lumholtzi_isoseq_coding.fasta --alignIntronMax 200000 | /img/gp/bin//compact_sam -max_intron 200000 -min_output_coverage 0.66 -min_query_len 21 -min_output_identity 0.9 | /img/gp/bin//samtools view -b' returned non-zero exit status 1.

#### qsub run_EGAPx_v0.3.1_HPC.sh test/EGAPx_v0.2/D_pulex/inputs_KAP106_ZQ_WW.txt
##### job 981468
ERROR
-> [W::hts_set_opt] Cannot change block size for this format
  samtools sort: failed to read header from "wrkarea/STAR.11001486083520O1Hzsx/D.pulex_KAP106_NMP_1.3.masked-KAP106_coding_transcripts_isoseq/D.pulex_KAP106_NMP_1.3.masked-KAP106_coding_transcripts_isoseq-Aligned.out.bam"
-> error [nextflow.exception.ProcessFailedException]: Process `egapx:rnaseq_short_plane:star:run_star (2)` terminated with an error exit status (3)
Nov-19 00:58:57.618 [TaskFinalizer-5] ERROR nextflow.processor.TaskProcessor - Error executing process > 'egapx:rnaseq_short_plane:star:run_star (2)'

#### qsub run_EGAPx_v0.3.1_HPC.sh test/EGAPx_v0.2/D_pulex/inputs_KAP106_ZQ_WW.fmt.txt
##### job 981479
ERROR
-> [W::hts_set_opt] Cannot change block size for this format
  samtools sort: failed to read header from "wrkarea/STAR.11202277877184Pmfy1D/D.pulex_KAP106_NMP_1.3.masked-KAP106_coding_transcripts_isoseq.fmt/D.pulex_KAP106_NMP_1.3.masked-KAP106_coding_transcripts_isoseq.fmt-Aligned.out.bam"

#### qsub run_EGAPx_v0.3.1_HPC.sh test/EGAPx_v0.2/D_pulex/inputs_KAP106_NCBI_WW.txt
##### job 981469
ERROR
-> [W::hts_set_opt] Cannot change block size for this format
  samtools sort: failed to read header from "wrkarea/STAR.11702967365056iMpBF3/GCA_028752575.1_ASM2875257v1_genomic-KAP106_coding_transcripts_isoseq/GCA_028752575.1_ASM2875257v1_genomic-KAP106_coding_transcripts_isoseq-Aligned.out.bam"

#### qsub run_EGAPx_v0.3.1_HPC.sh test/EGAPx_v0.2/D_pulex/inputs_KAP106_NCBI_WW.fmt.txt
##### job 981480
ERROR
-> [W::hts_set_opt] Cannot change block size for this format
  samtools sort: failed to read header from "wrkarea/STAR.116057435584ejT67Y/GCA_028752575.1_ASM2875257v1_genomic-KAP106_coding_transcripts_isoseq.fmt/GCA_028752575.1_ASM2875257v1_genomic-KAP106_coding_transcripts_isoseq.fmt-Aligned.out.bam"

#### qsub run_EGAPx_v0.3.1_HPC.sh test/EGAPx_v0.2/D_pulex/inputs_KAP106_NCBI_SRA.txt
##### job 981470
ERROR
-> [W::hts_set_opt] Cannot change block size for this format
  samtools sort: failed to read header from "wrkarea/STAR.22201597486528plBP4W/GCA_028752575.1_ASM2875257v1_genomic-SRR17282827/GCA_028752575.1_ASM2875257v1_genomic-SRR17282827-Aligned.out.bam"

#### qsub run_EGAPx_v0.3.1_HPC.sh test/EGAPx_v0.2/D_pulex/inputs_KAP106_ZQ_SRA.txt
##### job 981471
ERROR
-> [W::hts_set_opt] Cannot change block size for this format
  samtools sort: failed to read header from "wrkarea/STAR.2220139032000gp3sjX/D.pulex_KAP106_NMP_1.3.masked-SRR17282827/D.pulex_KAP106_NMP_1.3.masked-SRR17282827-Aligned.out.bam"

#### Protein tests

#### qsub run_EGAPx_v0.3.1_HPC.sh test/EGAPx_v0.2/D_pulex/inputs_KAP106_ZQ_ZQ_WW.txt
##### job 981472
ERROR
-> [W::hts_set_opt] Cannot change block size for this format
  samtools sort: failed to read header from "wrkarea/STAR.11403162346944Z6V3qQ/D.pulex_KAP106_NMP_1.3.masked-KAP106_coding_transcripts_isoseq/D.pulex_KAP106_NMP_1.3.masked-KAP106_coding_transcripts_isoseq-Aligned.out.bam"

#### qsub run_EGAPx_v0.3.1_HPC.sh test/EGAPx_v0.2/D_pulex/inputs_KAP106_ZQ_ZQ_WW.fmt.txt
##### job 981481
ERROR
-> [W::hts_set_opt] Cannot change block size for this format
  samtools sort: failed to read header from "wrkarea/STAR.11603239368128M97yJZ/D.pulex_KAP106_NMP_1.3.masked-KAP106_coding_transcripts_isoseq.fmt/D.pulex_KAP106_NMP_1.3.masked-KAP106_coding_transcripts_isoseq.fmt-Aligned.out.bam"

#### qsub run_EGAPx_v0.3.1_HPC.sh test/EGAPx_v0.2/D_pulex/inputs_KAP106_ZQ_ZQ_SRA.txt
##### job 981473
ERROR
-> [W::hts_set_opt] Cannot change block size for this format
  samtools sort: failed to read header from "wrkarea/STAR.22502089457088srqrFf/D.pulex_KAP106_NMP_1.3.masked-SRR17282827/D.pulex_KAP106_NMP_1.3.masked-SRR17282827-Aligned.out.bam"


## EGAPx v0.2

#### qsub run_EGAPx_v0.2_HPC.sh test/EGAPx_v0.3/D_pulex/inputs_KAP4_NCBI_small.txt
##### job 1026416
CDS          127684
exon         138413
gene         18152
mRNA         17787
transcript   2549 

#### qsub run_EGAPx_v0.2_HPC.sh test/EGAPx_v0.3/D_pulex/inputs_KAP4_NCBI_medium.txt
##### job 1026417
CDS          137446
exon         153191
gene         19301
mRNA         18721
transcript   3795

#### qsub run_EGAPx_v0.2_HPC.sh test/EGAPx_v0.3/D_pulex/inputs_KAP4_NCBI_ms.txt
##### job 1026612
CDS          137622
exon         153365
gene         19300
mRNA         18719
transcript   3795

#### qsub run_EGAPx_v0.2_HPC.sh test/EGAPx_v0.3/D_pulex/inputs_KAP4_NCBI_SRA_small.txt
##### job 1026399
CDS          127684
exon         138413
gene         18152
mRNA         17787
transcript   2549

#### qsub run_EGAPx_v0.2_HPC.sh test/EGAPx_v0.3/D_pulex/inputs_KAP4_NCBI_SRA_medium.txt
##### job 1026400
CDS          137622
exon         153365
gene         19300
mRNA         18719
transcript   3795

### KAP106 NCBI, ZQ, & WW tests

#### qsub run_EGAPx_v0.2_HPC.sh D_pulex/inputs_KAP106_NCBI_SRA.txt 
##### job 980812
Waiting for file transfers to complete (1 files)
Completed at: 09-Dec-2024 21:50:27
Duration    : 7d 12h 8m 20s
CPU hours   : 18'643.2 (39.2% cached)
Succeeded   : 1'399
Cached      : 61
Failed      : 4

#### qsub run_EGAPx_v0.2_HPC.sh D_pulex/inputs_KAP106_NCBI_SRA_WW.txt 
##### job 980814
COMPLETED

#### qsub run_EGAPx_v0.2_HPC.sh D_pulex/inputs_KAP106_ZQ_SRA.txt 
##### job 980815
COMPLETED

#### qsub run_EGAPx_v0.2_HPC.sh D_pulex/inputs_KAP106_ZQ_SRA_WW.txt 
##### job 980816
COMPLETED

#### qsub run_EGAPx_v0.2_HPC.sh D_pulex/inputs_KAP4_NCBI_SRA.txt 
##### job 980817
COMPLETED

#### qsub run_EGAPx_v0.2_HPC.sh D_pulex/inputs_KAP4_NCBI_SRA_WW.txt 
##### job 980818
KILLED


### KAP106 Isoseq test sets

#### Primary tests

#### qsub run_EGAPx_v0.2_HPC.sh test/EGAPx_v0.2/D_lumholtzi/inputs_lumholtzi_WW_RNA.txt
##### job 977147
SUCCEEDED
CDS          195421
exon         279971
gene         17462
mRNA         22527
transcript   9112

#### qsub run_EGAPx_v0.2_HPC.sh test/EGAPx_v0.2/D_lumholtzi/inputs_lumholtzi_WW.txt
##### job 956395
ERROR 
-> ERROR ~ Error executing process > 'egapx:rnaseq_short_plane:star:run_star (1)'
-> [E::sam_parse1] CIGAR and query sequence are of different length
  [W::sam_read1_sam] Parse error at line 53
  samtools view: error reading file "-"
  Traceback (most recent call last):
    File "/img/gp/bin/star-with-filter", line 32, in <module>
      subprocess.run(main_cmd, shell=True, check=True)
    File "/usr/lib/python3.8/subprocess.py", line 516, in run
      raise CalledProcessError(retcode, process.args,
  subprocess.CalledProcessError: Command 'set -euo pipefail; /img/gp/bin//../third-party/STAR/bin/Linux_x86_64/STAR --alignSJoverhangMin 8 --outFilterMultimapNmax 200 --outFilterMismatchNmax 50 --runThreadN 16 --genomeLoad NoSharedMemory --outSAMtype SAM --outSAMattributes NH HI AS nM NM MD jM jI XS MC --outSAMprimaryFlag AllBestScore --outFilterMultimapScoreRange 50 --seedSearchStartLmax 15 --limitOutSAMoneReadBytes 1000000 --outSJtype None --genomeDir D.lumholtzi_format.index --outFileNamePrefix wrkarea/STAR.1210575948224L4T8cm/D.lumholtzi_format-D.lumholtzi_isoseq_coding/D.lumholtzi_format-D.lumholtzi_isoseq_coding- --outStd SAM --readFilesIn ./D.lumholtzi_isoseq_coding.fasta --alignIntronMax 200000 | /img/gp/bin//compact_sam -max_intron 200000 -min_output_coverage 0.66 -min_query_len 21 -min_output_identity 0.9 | /img/gp/bin//samtools view -b' returned non-zero exit status 1.

#### qsub run_EGAPx_v0.2_HPC.sh test/EGAPx_v0.2/D_pulex/inputs_KAP106_ZQ_WW.txt
##### job 917382 
not formatted D obtusa WW data (oops)
##### job 925025
ERROR -> Plus 10 more processes waiting for tasks…
##### job 954911
SUCCEEDED
CDS          127742
exon         138430
gene         17947
mRNA         17642
transcript   2484

#### qsub run_EGAPx_v0.2_HPC.sh test/EGAPx_v0.2/D_pulex/inputs_KAP106_NCBI_WW.txt
##### job 925026
ERROR -> Plus 10 more processes waiting for tasks…
##### job 954912
SUCCEEDED
CDS          127742
exon         138430
gene         17947
mRNA         17642
transcript   2484

#### qsub run_EGAPx_v0.2_HPC.sh test/EGAPx_v0.2/D_pulex/inputs_KAP106_NCBI_SRA.txt
##### job 917475 -> non test data subset (oops)
COMPLETED
##### job 925027
ERROR -> ERROR ~ Error executing process > 'egapx:rnaseq_short_plane:rnaseq_collapse:generate_jobs'
##### job 954913
SUCCEEDED
CDS          127742
exon         138430
gene         17947
mRNA         17642
transcript   2484

#### qsub run_EGAPx_v0.2_HPC.sh test/EGAPx_v0.2/D_pulex/inputs_KAP106_ZQ_SRA.txt
##### job 917476 -> non test data subset (oops)
COMPLETED
##### job 925028
ERROR -> ERROR ~ Error executing process > 'egapx:rnaseq_short_plane:rnaseq_collapse:generate_jobs'
##### job 954914
SUCCEEDED
CDS          127742
exon         138430
gene         17947
mRNA         17642
transcript   2484

#### Protein tests

#### qsub run_EGAPx_v0.2_HPC.sh inputs_KAP106_ZQ_ZQ_SRA.txt
##### job 917381 -> full data set
COMPLETED
##### job 954915
The /scratch365/ebrooks5/multi_genome_project/EGAPx_v0.2/tested_november2024/ directory already exsists... please remove before proceeding.

#### qsub run_EGAPx_v0.2_HPC.sh test/EGAPx_v0.2/D_pulex/inputs_KAP106_ZQ_ZQ_WW.txt
##### job 917383 
ERROR
not formatted D obtusa WW data (oops) 
samtools sort: failed to read header from "wrkarea/STAR.1180121497024erNMJ5/D.pulex_KAP106_NMP_1.3.masked-D.obtusa_isoseq_coding/D.pulex_KAP106_NMP_1.3.masked-D.obtusa_isoseq_coding-Aligned.out.bam"
##### job 917396
not formatted WW data
ERROR -> Plus 10 more processes waiting for tasks…
##### job 925029
ERROR -> Plus 10 more processes waiting for tasks…
##### job 954916
SUCCEEDED
CDS          137123
exon         147254
gene         20378
mRNA         19860
transcript   2563

#### qsub run_EGAPx_v0.2_HPC.sh test/EGAPx_v0.2/D_pulex/inputs_KAP106_ZQ_ZQ_SRA.txt
##### job 917477 -> non test data subset (oops)
COMPLETED
##### job 925030
ERROR -> ERROR ~ Error executing process > 'egapx:rnaseq_short_plane:rnaseq_collapse:generate_jobs'
##### job 954917
SUCCEEDED
CDS          137123
exon         147254
gene         20378
mRNA         19860
transcript   2563


### ZQ & WW test sets

#### WW Isoseq file formatting
```
for i in /scratch365/ebrooks5/multi_genome_project/data/Daphnia_RNA_assembly/isoseq/*.fasta; do echo $i; newName=$(echo $i | sed "s/\.fasta/.fmt.fasta/g" | sed "s/Daphnia\_RNA\_assembly\/isoseq/Daphnia_RNA_assembly_formatted/g"); cat $i | cut -d"|" -f1 > $newName; done
```

#### qsub run_EGAPx_v0.2_HPC.sh test/EGAPx_v0.2/D_ambigua/inputs_D_ambigua_2_ZQ_WW.txt 
##### job 915611 -> the output directory was not tagged with "\_test"
formatted WW data 
no error, no output gff either 
##### job 925031
ERROR -> Plus 42 more processes waiting for tasks…
##### job 954920

#### qsub run_EGAPx_v0.2_HPC.sh test/EGAPx_v0.2/D_ambigua/inputs_D_ambigua_2_ZQ_ZQ_WW.txt
##### job 925032
ERROR -> Plus 42 more processes waiting for tasks…
##### job 954921

#### qsub run_EGAPx_v0.2_HPC.sh test/EGAPx_v0.2/D_obtusa/inputs_D_obtusa_2_ZQ_ZQ_trimmed.txt
##### job 925033
SUCCEEDED
##### job 954922

#### qsub run_EGAPx_v0.2_HPC.sh test/EGAPx_v0.2/D_obtusa/inputs_D_obtusa_2_ZQ_WW.txt
##### job 925041
ERROR -> Plus 42 more processes waiting for tasks…
##### job 954923

#### qsub run_EGAPx_v0.2_HPC.sh test/EGAPx_v0.2/D_obtusa/inputs_D_obtusa_2_ZQ_ZQ_WW.txt
##### job 925034
ERROR -> Plus 42 more processes waiting for tasks…
##### job 954924

#### qsub run_EGAPx_v0.2_HPC.sh test/EGAPx_v0.2/D_pulicaria/inputs_LK16_ZQ_ZQ_trimmed.txt
##### job 917384
ERROR ->   [ERRO] stat 18CRep6_TTAGGC_L002_R1.concat.fmt.fa: no such file or directory
##### job 925036
ERROR -> yaml.scanner.ScannerError: sequence entries are not allowed here
##### job 954925


### NCBI test set

#### qsub run_EGAPx_v0.2_HPC.sh inputs_KAP4_NCBI.txt
##### job 747604 
KILLED 
NOTE: Process `egapx:rnaseq_short_plane:star:run_star (180)` terminated with an error exit status (1) -- Execution is retried (1)

#### qsub run_EGAPx_v0.2_HPC.sh inputs_KAP106.txt
##### job 747606 
KILLED 
NOTE: Process `egapx:rnaseq_short_plane:star:run_star (174)` terminated with an error exit status (3) -- Execution is retried (1)

#### qsub run_EGAPx_v0.2_HPC.sh inputs_LK16_NCBI.txt
##### job 773689 
KILLED 
ERROR ~ Unable to re-submit task `egapx:rnaseq_short_plane:bam2asn:convert (376)`

#### qsub run_EGAPx_v0.2_HPC.sh inputs_SC_F0-13Bv2.txt
##### job 773695 
KILLED 
NOTE: Process `egapx:rnaseq_short_plane:fetch_sra_fasta:run_fetch_sra_fasta (85)` terminated with an error exit status (23) -- Execution is retried (1) 
ERROR ~ Unable to re-submit task `egapx:rnaseq_short_plane:fetch_sra_fasta:run_fetch_sra_fasta (85)`

#### qsub run_EGAPx_v0.2_HPC.sh inputs_NIES_NCBI.txt
##### job 773797 
KILLED 
NOTE: Process `egapx:rnaseq_short_plane:fetch_sra_fasta:run_fetch_sra_fasta (58)` terminated with an error exit status (23) -- Execution is retried (1)


### SRA test set

#### qsub run_EGAPx_v0.2_HPC.sh inputs_KAP4_SRA.txt
##### job 800519 
ERROR ~ Error executing process > 'egapx:rnaseq_short_plane:star:run_star (124)' 
Process `egapx:rnaseq_short_plane:star:run_star` input file name collision -- There are multiple input files for each of the following file names: SRR17282827_1.fasta
##### job 808160 
SUCCEEDED

#### qsub run_EGAPx_v0.2_HPC.sh inputs_KAP106_ZQ_SRA.txt
##### job 800520 
ERROR ~ Error executing process > 'egapx:rnaseq_short_plane:star:run_star (124)' 
Process `egapx:rnaseq_short_plane:star:run_star` input file name collision -- There are multiple input files for each of the following file names: SRR17282827_1.fasta
##### job 808161 
SUCCEEDED

#### qsub run_EGAPx_v0.2_HPC.sh inputs_LK16_SRA.txt
##### job 800521 
SUCCEEDED

#### qsub run_EGAPx_v0.2_HPC.sh inputs_NIES_SRA.txt
##### job 800522 
SUCCEEDED

#### qsub run_EGAPx_v0.2_HPC.sh inputs_SC_F1_1A_NCBI_SRA.txt
##### job 823853 
SUCCEEDED

#### qsub run_EGAPx_v0.2_HPC.sh inputs_KAP106_NCBI_SRA.txt
##### job 821229 
KILLED 
/scratch365/ebrooks5/multi_genome_project/EGAPx_v0.2/D_pulex_KAP106_NCBI_SRA/accept.gff

#### qsub run_EGAPx_v0.2_HPC.sh inputs_LK16_NCBI_trimmed.txt
##### job 821670 
SUCCEEDED

#### qsub run_EGAPx_v0.2_HPC.sh inputs_CSIRO_1_NCBI_SRA.txt
##### job 854120 
ERROR? -> gff was produced but statistics were not output
Waiting for file transfers to complete (1 files)
Completed at: 31-Oct-2024 18:18:21
Duration    : 13h 21s
CPU hours   : 43'391.9 (98.1% cached)
Succeeded   : 363
Cached      : 427

#### qsub run_EGAPx_v0.2_HPC.sh inputs_Xinb3_NCBI_SRA.txt
##### job 854121 
ERROR
EXITING because of FATAL ERROR in reads input: Lread of the pair = 856   while DEF_readSeqLengthMax=650
 Read Name=>SRR2671679.1952.2
 SOLUTION: increase DEF_readSeqLengthMax in IncludeDefine.h and re-compile STAR

#### qsub run_EGAPx_v0.2_HPC.sh inputs_LRV0_1_NCBI_SRA.txt
##### job 854182 
SUCCEEDED

#### qsub run_EGAPx_v0.2_HPC.sh inputs_SK_NCBI_SRA.txt
##### job 854669 
SUCCEEDED

#### qsub run_EGAPx_v0.2_HPC.sh inputs_FS6_NCBI_trimmed.txt
##### job 892539 
SUCCEEDED

#### qsub run_EGAPx_v0.2_HPC.sh inputs_FS6_ZQ_trimmed.txt 
inputs_D_obtusa_2_ZQ_trimmed.txt
##### job 892541 
ERROR 
typo in yaml file
##### job 915292 
SUCCEEDED


## EGAPx v0.1

#### qsub run_EGAPx_HPC.sh inputs_KAP4_NCBI.txt
##### job 694733 
DONE 
ERROR ~ Error executing process > 'egapx:star:exec (111)'
Wallclock Time   = 10:23:29:22

#### qsub run_EGAPx_HPC.sh inputs_KAP106.txt
##### job 699252 
DONE 
ERROR ~ Error executing process > 'egapx:star:exec (107)'
Wallclock Time   = 8:15:36:18

#### qsub run_EGAPx_HPC.sh inputs_LK16_NCBI.txt
##### job 773663 
KILLED 
Error: (CSQLITE_Exception::eStmtStep) Error from sqlite3: [778] disk I/O error

#### qsub run_EGAPx_HPC.sh inputs_SC_F0-13Bv2.txt
##### job 773672 
DONE 
curl: (52) Empty reply from server
Wallclock Time   = 4:16:23:22
Process `egapx:fetch_sra_fasta:run_fetch_sra_fasta (69)` terminated with an error exit status (52)

#### qsub run_EGAPx_HPC.sh inputs_NIES_NCBI.txt
##### job 773796 
KILLED 
ERROR ~ Error executing process > 'egapx:run_get_hmm'
